### PR TITLE
[FR] add missing answers in sentences and tests

### PR DESCRIPTION
**Pull request by flexy2dd**

### CODE REVIEW

File: responses/fr/HassTurnOff.yaml [hunk 1]
```diff
@@ -5,6 +5,8 @@ responses:
       default: "{{ slots.name }} éteint"
       lights_area: "Lumières de {{ slots.area }} éteintes"
       fans_area: "Ventilateurs de {{ slots.area }} éteints"
+      light_all: "Éteindre toutes les lumieres"
```
Comment (bemble) [inline responses/fr/HassTurnOff.yaml:8]:
```suggestion
      light_all: "Toutes les lumières sont éteintes"
```

File: responses/fr/HassTurnOff.yaml [hunk 2]
```diff
@@ -5,6 +5,8 @@ responses:
...
+      fan_all: "Éteindre tous les ventilateurs"
```
Comment (bemble) [inline responses/fr/HassTurnOff.yaml:9]:
```suggestion
      fan_all: "Tous les ventilateurs sont éteints"
```

File: responses/fr/HassTurnOn.yaml [hunk 3]
```diff
@@ -5,6 +5,9 @@ responses:
       default: "{{ slots.name }} allumé"
       lights_area: "Lumières de {{ slots.area }} allumées"
       fans_area: "Ventilateurs de {{ slots.area }} allumés"
+      light_all: "Allume toutes les lumières"
+      fan_all: "Allume tous les ventilateurs"
       cover: "{{ slots.name }} ouvert"
       cover_area: "Ouvrants de {{ slots.area }} ouverts"
       cover_device_class: "{{ slots.device_class }} ouvert"
+      cover_all: "Ouvre tous les ouvrants de {{ slots.area }}"
```
Comment (bemble) [inline responses/fr/HassTurnOn.yaml:13]: I'm not that happy with "ouvrant" for "cover", but I don't have any better proposal.

File: tests/fr/fan_HassTurnOff.yaml [hunk 4]
```diff
@@ -9,3 +9,4 @@ tests:
         area: salon
         domain: fan
         name: all
+    response: "Éteindre tous les ventilateurs"
```
Comment (bemble) [inline tests/fr/fan_HassTurnOff.yaml:12]:
```suggestion
    response: "Tous les ventilateurs sont éteints"
```

File: tests/fr/homeassistant_HassTurnOff.yaml [hunk 5]
```diff
@@ -9,27 +9,31 @@ tests:
       slots:
         name: lumière du plafond
         area: chambre
+    response: "lumière du plafond éteint"
```
Comment (bemble) [inline tests/fr/homeassistant_HassTurnOff.yaml:12]: To be compliant with the style guide, response should not contain the entity name. For now, I would correct the test, but the reply should be `lumière éteinte`

```suggestion
    response: "lumière du plafond éteinte"
```

File: tests/fr/homeassistant_HassTurnOff.yaml [hunk 6]
```diff
@@ -9,27 +9,31 @@ tests:
...
   - sentences:
       - éteins le ventilateur de plafond
       - arrête le ventilateur de plafond
     intent:
       name: HassTurnOff
       slots:
         name: ventilateur de plafond
+    response: "ventilateur de plafond éteint"
```
Comment (bemble) [inline tests/fr/homeassistant_HassTurnOff.yaml:20]: Same as previous comment, the reply should just be `ventilateur éteint`

**Review by bemble — CHANGES_REQUESTED** [review]
The answers for `fan_all` and `light_all` have to be changed.
I just saw that the french translation is not fully compliant with the [style-guide](https://developers.home-assistant.io/docs/voice/intent-recognition/style-guide), we need to create a separate PR for that.

File: responses/fr/HassTurnOn.yaml (hunk 3 above)
Comment (flexy2dd) [reply responses/fr/HassTurnOn.yaml:13]: Yes agree, i did not find either
it is difficult because it can be the windows, the shutters, etc

**Review by bemble — APPROVED** [review]

**Pull request merged**